Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -12,157 +12,106 @@ public CredentialsManager()
-            } while (string.IsNullOrEmpty(password.Trim()));
-
-            return new Credentials(username, password);
-        }
-
-
-
         private bool IsNameRegistered(string username)
         {
-            var user = GetUser(username);
+            var credentials = GetStoredCredentials(username);
 
-            if (user.Username == null)
+            if (credentials.Username == null)
             {
                 return false;
             }
 
             return true;
         }
 
-        private bool AuthenticateUser(Credentials user)
+        public bool Login(Credentials credentials)
         {
-            var storedUser = GetUser(user.Username);
+            var storedCredentials = GetStoredCredentials(credentials.Username);
 
-            if (storedUser.Username == null || storedUser.Password != new Cryptography().Encode(user.Password))
+            if (storedCredentials.Username == null || storedCredentials.Password != new Cryptography().Encode(credentials.Password))
             {
                 return false;
             }
 
             return true;
         }
-
-        private Credentials GetUser(string username)
+               
+        private Credentials GetStoredCredentials(string username)
         {
-            foreach (var user in GetCredentials())
+            foreach (var credentials in ReadCredentials())
             {
-                if (user.Username == username)
+                if (credentials.Username == username)
                 {
-                    return user;
+                    return credentials;
                 }
             }
 
             return new Credentials();
         }
 
-        private List<Credentials> GetCredentials()
+        private List<Credentials> ReadCredentials()
         {
-            var users = new List<Credentials>();
+            var credentialsList = new List<Credentials>();
 
-            var credentials = File.ReadAllLines(CredentialsPath);
+            var readCredentials = File.ReadAllLines(CredentialsPath);
 
-            foreach (var credential in credentials)
+            foreach (var credentials in readCredentials)
             {
-                if (!TryCredentialsParse(credential, out var user))
+                if (!TryCredentialsParse(credentials, out var parsedCredentials))
                 {
-                    throw new InvalidDatabaseException("Unpexpected data found in credential database.");
+                    throw new InvalidDatabaseException("Unpexpected data found in credentials database.");
                 }
 
-                users.Add(user);
+                credentialsList.Add(parsedCredentials);
             }
 
-            return users;
+            return credentialsList;
         }
 
-        private bool TryCredentialsParse(string credential, out Credentials users)
+        private bool TryCredentialsParse(string credentials, out Credentials parsedCredentials)
```

Comment:
This should be inside the Credentials class. Remember the TryParse pattern?
It shoudl be Credentials.TryParse(...)